benches/benches/bevy_ecs/index/mod.rs

```diff
@@ -0,0 +1,38 @@
+mod index_iter_indexed;
+mod index_iter_naive;
+mod index_update_indexed;
+mod index_update_naive;
+
+use criterion::{criterion_group, Criterion};
+
+criterion_group!(benches, index_iter, index_update,);
+
+fn index_iter(c: &mut Criterion) {
+    let mut group = c.benchmark_group("index_iter");
+    group.warm_up_time(core::time::Duration::from_millis(500));
+    group.measurement_time(core::time::Duration::from_secs(4));
```

Comment:
Is there a specific reason you chose these times, which are smaller than the default? `criterion` by default has a warm-up time of 3 seconds and a measurements time of 5 seconds.